src/main/java/com/wfector/command/CommandRunner.java

```diff
@@ -91,7 +105,9 @@ public boolean onCommand(final CommandSender sender, Command cmd, String label,
                                     if (userId != null) {
                                         cleaner.cleanUser(userId);
                                     } else {
-                                        sender.sendMessage(ChatColor.RED + args[i + 1] + " is not a valid username/uuid input for " + param.getUsage() + "!");
+                                        if(plugin.getMessage("invalid-username") != null) {
+                                            sender.sendMessage(plugin.getMessage("invalid-username").replace("{typo}", args[i + 1]).replace("{usage}", param.getUsage()));
```

Comment:
Would be good if we could use a better way to replace placeholders in messages. Something like `getMessage("invalid-username", "type", args[i + 1])` via a [`getMessage(String key, String... replacements)` method](https://github.com/Minebench/Guilds/blob/master/src/main/java/io/github/apfelcreme/Guilds/GuildsConfig.java#L415)?